### DISCUSSION TITLE
v2.21.0: Pokémon TCG Pocket & Abbreviation support

### DISCUSSION THREAD

**Discussion by Aviortheking** [tcgdex/cards-database#574] [Announcements]
new API Update `2.21.0` :

First new Asian Sets were added to the database to get closer to being up to date with the current releases :
- `SV08` in Japanese and Chinese
- `svLN` in Japanese
- `svLS` in Japanese
- `SV7` in Japanese and Chinese
- `SV7a` in Japanese and Chinese
- `SV7s` in Indonesian and Thai

A New "Serie" (*the word is being stretched a bit*) : `Pokémon TCG Pocket`. It will contains sets that are out on the TCG Pocket app
The following set was added to it :
- Genetic Apex `A1` *In english only for the moment being sry*
- I'll add the current promo `PROMO-A` later this week

New Field :
- Abbreviation, now every sets will have an `abbreviation` field that is linked to the official TCG documentation, Also an unofficial for other languages is also in the same field.

**note: Documentation will be updated tomorrow, it's too late for me :face_holding_back_tears: **

*As always, big thanks to the Sponsors of the project that help me work on the project https://github.com/sponsors/tcgdex :heart:*

**Comment by wsgtcyx** [comment]
Looking forward to PROMO-A of TCG Pocket!   Thanks for your quick updating !  I Sposor tcgdex~!
